test/test_mps.jl

```diff
@@ -117,3 +117,74 @@ using ITensors,
   replaceBond!(psi,1,phi)
   @test tags(linkindex(psi,1)) == bondindtags
 end
+
+# Helper function for making MPS
+function basicRandomMPS(N::Int;dim=4)
+  sites = SiteSet(N,2)
+  M = MPS(sites)
+  links = [Index(dim,"n=$(n-1),Link") for n=1:N+1]
```

Comment:
I know this is just a test function, but I think we should try to be consistent with link indices having the tags "Link, l=$n". It is useful for site indices to have "n=$n" tags and link indices to have "l=$n" tags so that to grab the right link of site `n`, you can just do `findindex(M[n],"l=$n")` and not have to also list the "Link" tag. (I think you know this, just writing it out since this was something that was inconsistent at some point in the C++ version and I kept seeing it pop up).